packages/tophide/tophide.1.0.4/opam

```diff
@@ -0,0 +1,11 @@
+opam-version: "1.2"
+maintainer: "andy.ray@ujamjar.com"
```

Comment:
I guess it doesn't really matter because the package is tiny. Also I'm very happy to have someone else's name listed rather than my own.